### PR TITLE
miscellaneous edits across the handbook

### PR DESCRIPTION
**Pull request by billwanjohi**
[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/miscellaneous-edits/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

File: docs/030-policies/security.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ CivicActions requires unique, strong passwords for every service that you log in
 
 Use both a unique, strong password (or multi-word passphrase) _and_ two-factor authentication (TFA or 2FA) to secure your password manager.
 
-CivicActions supplies all employees with a YubiKey for two-factor authentication, and we require two-factor authentication for access to your password manager, the CivicActions GSuite of applications, and OAuth-authenticated apps such as GitHub and GitLab. You can also use second factor authentication apps such as Authy and Duo, and in fact, for backup reasons we suggest using more than one second factor.
+CivicActions supplies all employees with a YubiKey for two-factor authentication, and we require two-factor authentication for access to your password manager, the CivicActions GSuite of applications, and OATH-authenticated apps such as GitHub and GitLab. You can also use second factor authentication apps such as Authy and Duo, and in fact, for backup reasons we suggest using more than one second factor.
```
Comment (billwanjohi) [inline docs/030-policies/security.md]: I messed this up myself in one of my first weeks here. :facepalm:

Comment (grugnog) [reply docs/030-policies/security.md]: @billwanjohi I don't think most team members would know what OAuth or OATH is, and I am not sure it matters here.
I think we could rephrase this whole sentence to be much more concise: "access to your password manager, CivicActions GSuite, GitHub and GitLab"?

Comment (grugnog) [reply docs/030-policies/security.md]: Everything else in this PR looks great - thank you :)

Comment (billwanjohi) [reply docs/030-policies/security.md]: OK, as suggested, I trimmed that.

**Review by mgifford — COMMENTED** [review]
These all make sense to me. Wasn't sure why we were excluding .html files with the .gitignore

I also wanted someone to look at the expense process and give better feedback than I could.

**Comment by billwanjohi** [comment]
@mgifford great question.

The background is that the repository does not currently have any html files committed, and it doesn't look like there is any near-term intention to add any, because the static site generator we use is configured to process markdown (.md) files to generate our html.

My specific impetus for adding this rule was that I did a local conversion of a few markdown files to html to preview some changes I was making before pushing them to GitHub. Once I do that, my local git utility will ask me whether I want to add and commit these files. Since they are derivative of the already-tracked markdown files, I'll never want to do so, so I'm squelching the suggestion for all files of this pattern by adding it to the gitignore file.

If at some later point we have uses for html, we can narrow this rule to particular patterns/directories/etc., but in the meantime, I like to assume that additional ignore rules make for a smoother experience for most people, particularly when they are just starting to engage with a repository. Less cognitive load.

**Review by grugnog — APPROVED** [review]

**Pull request merged**